File: PYTHON/01 - LINEAR/integer and float in given  ranges/file.py
# From the math module, import random, randint
# the random() returns random float 
# the randint returns random integer
from random import randint,random

# collect upper and lower limit of the range
print("Range of integers")
intMin = int(input("enter minimum:"))
intMax = int(input("enter Maximum:"))

n = randint(intMin,intMax)

print("random int:",n)

# collect upper and lower limit of the range
print("Range of floats")
floatMin = float(input("enter minimum:"))
floatMax = float(input("enter Maximum:"))

n = random() * (floatMax-floatMin) + floatMin

print("random float:",n)

File: JAVASCRIPT/01 - LINEAR/integer and float in given  ranges/file.js
//  collect upper and lower limit of the range
// Math.random() generates random number from 
alert("enter range for integers:")
const intMin = parseInt(prompt("enter minimum: "))
const intMax = parseInt(prompt("enter Maximum: "))

let n = Math.floor((Math.random() + 0.2) *  (intMax-intMin) + intMin)

console.log("random int:",n)
alert("enter range for floats:")
//  collect upper and lower limit of the range
floatMin = parseFloat(prompt("enter minimum: "))
floatMax = parseFloat(prompt("enter Maximum: "))

n = Math.random() *  (floatMax-floatMin) + floatMin

console.log("random float:",n)

